projects/angular/src/forms/common/wrapped-control.ts

```diff
@@ -55,30 +55,34 @@ export class WrappedFormControl<W extends DynamicWrapper> implements OnInit, DoC
   private markControlService: MarkControlService;
   private containerIdService: ContainerIdService;
   private _containerInjector: Injector;
-  private differs: KeyValueDiffers;
-  private differ: KeyValueDiffer<any, any>;
-  private secondaryDiffer: KeyValueDiffer<any, any>;
-  private ngControl: NgControl | null;
+  private kvDiffers: KeyValueDiffers;
+  private differs: KeyValueDiffer<any, any>[];
```

Comment:
todo: Having 2 parallel arrays that should be syncronized by index may not be the best approach.
We may want to rework this to [{control, differ}] format.